### PR TITLE
 logs verbosity validation for kube-rbac containers

### PR DESCRIPTION
**Pull request by miyadav**
Here are validation results - https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/227715/console

This is to cover  -https://bugzilla.redhat.com/show_bug.cgi?id=1907380

@sunzhaohua2  @jhou1  PTAL

### CODE REVIEW

**Comment by sunzhaohua2** [comment]
@miyadav can we also add these containers? I just checked seems these still has excessive verbosity.
oc logs -f machine-api-operator-cfcc58f99-gxxz6 -c kube-rbac-proxy
oc logs -f cluster-autoscaler-operator-5d7d4cc68d-rm756 -c kube-rbac-proxy

File: features/machine/machine_misc.feature [hunk 1]
```diff
@@ -106,3 +106,27 @@ Feature: Machine misc features testing
...
+  # @author miyadav@redhat.com
+  # @case_id OCP-37744
+  @admin
+  Scenario Outline: kube-rbac-proxy should not expose tokens, have excessive verbosity
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    Then I use the "<project>" project
+
+    Given a pod becomes ready with labels:
+      | api=clusterapi| <labels>  |
```
Comment (sunzhaohua2) [inline features/machine/machine_misc.feature:119]: `| api=clusterapi | <labels> |`

File: features/machine/machine_misc.feature [hunk 2]
```diff
@@ -106,3 +106,27 @@ Feature: Machine misc features testing
       | o             | json                         |
     Then the expression should be true>  @result[:parsed]['webhooks'][0]['clientConfig']['service']['port'] == 443
     """
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-37744
+  @admin
+  Scenario Outline: kube-rbac-proxy should not expose tokens, have excessive verbosity
+    Given I have an IPI deployment
```
Comment (jhou1) [inline features/machine/machine_misc.feature:114]: Can this be tested on UPI?

File: features/machine/machine_misc.feature (hunk 1 above)
Comment (miyadav) [reply features/machine/machine_misc.feature:119]: Thanks a lot @sunzhaohua2 , added those , it seems both are bug 
  [07:42:49] INFO> Exit Status: 0
      | resource_name | <%= pod.name %> |
      | c             | kube-rbac-proxy |
        Then the output should not contain:                                                       # features/step_definitions/common.rb:33
      | Response Headers |
      pattern found: Response Headers (RuntimeError)
will follow up on those ..

File: features/machine/machine_misc.feature (hunk 2 above)
Comment (miyadav) [reply features/machine/machine_misc.feature]: yes works on UPI

File: features/machine/machine_misc.feature [hunk 3]
```diff
@@ -106,3 +106,28 @@ Feature: Machine misc features testing
...
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-37744
+  @admin
+  Scenario Outline: kube-rbac-proxy should not expose tokens, have excessive verbosity
+    Given I switch to cluster admin pseudo user
+    Then I use the "<project>" project
+
+    Given a pod becomes ready with labels:
+      | api=clusterapi| <labels>  |
```
Comment (sunzhaohua2) [inline features/machine/machine_misc.feature:118]: change this line to `| api=clusterapi | <labels> |`

Comment (miyadav) [reply features/machine/machine_misc.feature]: done and squashed commits

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**